trade-finance-manager-api/src/v1/emails/amendments/manual-amendment-decision-email-variables.js

```diff
@@ -0,0 +1,123 @@
+const { AMENDMENT_UW_DECISION, AMENDMENT_TYPE } = require('../../../constants/deals');
+
+// email variables if amendment includes approved with conditions but not declined
+const approvedWithConditionsDecision = (amendmentVariables) => {
+  const { amendment, dealSnapshot, user } = amendmentVariables;
+  const { ukefFacilityId, ukefDecision } = amendment;
+  const { exporter, ukefDealId, bankInternalRefName } = dealSnapshot;
+  const { firstname, surname } = user;
+
+  return {
+    recipientName: `${firstname} ${surname}`,
+    ukefFacilityId,
+    exporterName: exporter.companyName,
+    bankReferenceNumber: bankInternalRefName,
+    ukefDealId,
+    conditions: ukefDecision.conditions,
+  };
+};
+
+// email variables if amendment is approved without conditions only
+const approvedWithoutConditionsDecision = (amendmentVariables) => {
+  const { amendment, dealSnapshot, user } = amendmentVariables;
+  const { ukefFacilityId } = amendment;
+  const { exporter, ukefDealId, bankInternalRefName } = dealSnapshot;
+  const { firstname, surname } = user;
+
+  return {
+    recipientName: `${firstname} ${surname}`,
+    ukefFacilityId,
+    exporterName: exporter.companyName,
+    bankReferenceNumber: bankInternalRefName,
+    ukefDealId,
+  };
+};
+
+// email variables if amendment is approved with conditions and declined
+const approvedWithConditionsDeclinedDecision = (amendmentVariables) => {
+  const { amendment, dealSnapshot, user } = amendmentVariables;
+  const { ukefFacilityId, ukefDecision } = amendment;
+  const { exporter, ukefDealId, bankInternalRefName } = dealSnapshot;
+  const { firstname, surname } = user;
+  const { conditions, declined, value } = ukefDecision;
+
+  const amendmentTypeApproved = value === AMENDMENT_UW_DECISION.APPROVED_WITH_CONDITIONS ? AMENDMENT_TYPE.VALUE : AMENDMENT_TYPE.COVER_END_DATE;
```

Comment:
Should be 
```
const amendmentTypeApproved = (value === AMENDMENT_UW_DECISION.APPROVED_WITH_CONDITIONS ? AMENDMENT_TYPE.VALUE : AMENDMENT_TYPE.COVER_END_DATE);

Reply:
we don't need the brackets actually because it's evaluating only one condition. ESlint would format it without the brackets as well